### PR TITLE
refactoring sideMenu design

### LINKED ISSUE #907: Implement new visual language in Menu
This is what it looks like right now:

editor - editi

We need to update it, as the turquoise colour is deprecated, some icons have been updated, and we generally need to make it look happier.

### PR DESCRIPTION
**Pull request by gvn**
Resolves #907 

R? @gesa

### CODE REVIEW

File: static/content/icons/back.svg [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="10px" height="11px" viewBox="0 0 10 11" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns">
+    <!-- Generator: Sketch 3.2.2 (9983) - http://www.bohemiancoding.com/sketch -->
+    <title>back</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="Editor" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" sketch:type="MSPage">
+        <g id="Menu" sketch:type="MSArtboardGroup" transform="translate(-10.000000, -81.000000)" fill="#64A8EE">
+            <g sketch:type="MSLayerGroup" transform="translate(0.000000, 19.000000)" id="Group">
+                <g sketch:type="MSShapeGroup">
```
Comment (gesa) [inline static/content/icons/back.svg]: Can we clean this up and get rid of all the `sketch:foo` attributes?

File: static/content/icons/back.svg [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+                    <g id="Rectangle-492-+-Share-+--3" transform="translate(0.000000, 43.000000)">
+                        <path d="M12.3772311,23.8529412 L20,23.8529412 L20,25.1470588 L12.3772311,25.1470588 L16.2119814,28.9818092 L15.2969021,29.8968885 L9.9936012,24.5935877 L10.0871889,24.5 L9.9936012,24.4064123 L15.2969021,19.1031115 L16.2119814,20.0181908 L12.3772311,23.8529412 Z" id="back"></path>
+                    </g>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (gesa) [inline static/content/icons/back.svg]: All three of these SVGs also need a newline

Comment (gesa) [inline static/content/icons/back.svg]: Four. all four. I can count.

File: components/sideMenu/index.less [hunk 3]
```diff
@@ -19,14 +19,14 @@
     .primary-menu {
         position: relative;
         transform: translateX(-100%);
-        background-color: @turquoise;
-        color: @white;
+        background-color: #ebeff7;
```
Comment (gesa) [inline components/sideMenu/index.less]: Does it make sense to make this color (and `#cedee9` below) variables? Or do you think this is the only place they'll be used?

**Comment by gesa** [comment]
This looks beautiful. A little boring SVG cleanup, then R+

File: components/sideMenu/index.less (hunk 3 above)
Comment (gvn) [inline components/sideMenu/index.less]: AFAIK these are unique to the menu.

Comment (gesa) [inline components/sideMenu/index.less]: 👍

**Pull request merged**